play/apps/snake/models.py

```diff
@@ -18,6 +18,10 @@ class Snake(models.Model):
     class Meta:
         app_label = 'snake'
 
+    def get_leaderboard_games(self):
+        from apps.game.models import Game
+        return Game.objects.filter(gamesnake__snake_id=self.id, is_leaderboard_game=True).order_by("-modified")[:5]
```

Comment:
should `5` here be a configurable number?

Reply:
at some point it could be, do we want to make that change before merging? Originally it was 10, but this added a lot of extra height on the listing page, I switched it to 5 to be more manageable. I imagine there will be a ui design and at that point would be a good time to make it say user configurable.

Reply:
Sounds good!